commons/ava_services/gecko_service_initializer_core.go

```diff
@@ -166,16 +168,16 @@ func (g GeckoServiceInitializerCore) GetStartCommand(mountedFileFilepaths map[st
 			avaDependencies = append(avaDependencies, service.(AvaService))
 		}
 
+		defaultStakers := ava_default_testnet.DefaultTestNet.DefaultStakers
 		socketStrs := make([]string, 0, len(avaDependencies))
-		for _, service := range avaDependencies {
+		bootstrapIds := make([]string, 0, len(avaDependencies))
+		for i, service := range avaDependencies {
 			socket := service.GetStakingSocket()
 			socketStrs = append(socketStrs, fmt.Sprintf("%s:%d", socket.GetIpAddr(), socket.GetPort().Int()))
-			if g.stakingTlsEnabled {
-				// We hardcode the first bootstrapper ID from the TLS identities in gecko_service_tls_identities
-				commandList = append(commandList, "--bootstrap-ids=" + STAKER_1_NODE_ID)
-				// We currently have one cert -> ID mapping so break the for loop here.
-				break
-			}
+			bootstrapIds = append(bootstrapIds, defaultStakers[i].NodeID)
```

Comment:
This will break if they specify > 5 bootstrapper IDs (which would be legit)

Reply:
adding check for length here